integration/extension-import/extension-import-test.ts

```diff
@@ -0,0 +1,13 @@
+import * as path from "node:path";
+import * as fs from "node:fs";
+import * as ts from "typescript";
+
+describe('extension-only-files', () => {
+  it("generate as external module", () => {
+    const generatedPath = path.join(__dirname, "extension.ts");
+    const generatedCode = fs.readFileSync(generatedPath, "utf8");
+    const source = ts.createSourceFile(generatedPath, generatedCode, ts.ScriptTarget.ES2018);
```

Comment:
I appreciate the test @jk-apple , but I think in this case our "lazy" approach of just checking in the `extension.ts` file will serve as a good-enough regression + probably resolve the CI error that's failing to find the file to assert against.

If you could push that up, that'd be great; appreciate the fix!

Reply:
Thanks for the help, pushed it.